README.md

```diff
@@ -14,13 +14,16 @@ Aleph node is based on a Substrate node where the default finality gadget (GRAND
 
 ### Installation
 
-1. Install the Substrate framework as described [here][substrate-installation], using the `nightly-2021-10-24` version of the rust toolchain (see the instruction at the bottom of the page)
-2. Clone this repository to your machine
-3. Run `cargo build --release -p aleph-node`
+1. Install `rustup` as described [here][rustup-instalation].
+2. Clone this repository to your machine and go to the top-level directory.
+3. This project is tied to a specific version of the rust toolchain defined in the `rust-toolchain` file. All `rustup` commands, including `cargo`, will respect this setting unless you tell them to do otherwise, so make sure that you have **not** set any [overrides][rustup-overrides] with higher priority.
+4. Run `rustup show` to install the toolchain.
```

Comment:
Wait, `rustup show` installs something? Doesn't it only list the version of rust you have installed?

Reply:
There's a four-year-old discussion about this behaviour with no conclusions so far: https://github.com/rust-lang/rustup/issues/1397

Reply:
Oh wow :face_with_spiral_eyes: